libs/user-activity/interface/index.ts

```diff
@@ -0,0 +1,9 @@
+export interface user_Activity {
```

Comment:
Kindly use the best practice to declare a interface name.

Reply:
Done